src/main/webapp/TodoMvc.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <title>Tessell • TodoMVC</title>
-    <script type="text/javascript" language="javascript" src="/todomvc/todomvc.nocache.js"></script>
+    <script type="text/javascript" language="javascript" src="todomvc/todomvc.nocache.js"></script>
```

Comment:
This shouldn't make a difference in how the port is interpreted; e.g. "/todomvc" does not mean "foo:80/todomvc", it just means "<whatever host+port>/todomvc". My assumption is that something else was wrong. (I run on :8080 locally as well and haven't had this problem.)